activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -293,7 +293,13 @@ def lease_connection
         lease.connection ||= checkout
       end
 
-      alias_method :connection, :lease_connection # TODO: deprecate
+      def connection
+        ActiveRecord.deprecator.warn(<<~MSG)
+          ConnectionPoool#connection is deprecated and will be removed
```

Comment:
Typo

Reply:
🤦 Thank you.

Reply:
Oh, I wasn't even meaning this. I just noticed that `Poool` doesn't look very correct. My bad, should have been more specific.

Reply:
Yes, I meant to fix that too, but I must be tired 😫